src/hw-monitor.h

```diff
@@ -266,7 +119,12 @@ namespace librealsense
         }
     };
 
-    std::string hwmon_error_string( command const &, hwmon_response e );
+    typedef int32_t hwmon_response;
+    class base_hwmon_response_handler { // base class for different product number to implement responses
```

Comment:
We don't have product numbers, you probably meant product ID, which also does not sound correct here. Rephrase to `products` or `product lines`